sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_models.py

```diff
@@ -0,0 +1,166 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+# pylint:disable=protected-access
+
+class KeyVaultPermission(object):
+    """Role definition permissions.
+
+    :ivar list[str] actions: allowed actions
+    :ivar list[str] not_actions: denied actions
```

Comment:
side note: not sure if this has already been a design discussion, but I feel like `denied_actions` sounds better than `not_actions`

Reply:
Sure, that's reasonable. I don't know whether there's been a discussion either, I just copied this from @christothes 😊

Reply:
I'm using denied actions as well.